### DISCUSSION TITLE
Original Xbox button icons?

### DISCUSSION THREAD

**Discussion by Kode-Z** [redphx/better-xcloud#353] [General]
What are the icon values for original Xbox control layouts? Like the white/black buttons.

**Comment by redphx** [comment]
I don't understand?

**Comment by Kode-Z** [comment]
![image](https://github.com/redphx/better-xcloud/assets/162533196/8650b723-10a6-470d-a4fa-ff3ee5102102)

Default touch layouts for classic Xbox games have bespoke icons for OG Xbox and 360 games. Like for Crimson Skies, there's the Black button, and the Duke controller's start button.

**Reply by Kode-Z** [reply]
![image](https://github.com/redphx/better-xcloud/assets/162533196/b2f46eaf-19a8-4965-b0d7-e1464286d6f5)

Here is Fable on 360.

**Comment by redphx** [comment]
It uses custom assets. Better xCloud doesn't support this yet.
